sharpd/sharp_zebra.c

```diff
@@ -129,6 +129,8 @@ static int interface_state_down(int command, struct zclient *zclient,
 	return 0;
 }
 
+struct timeval t_start;
+struct timeval t_end;
```

Comment:
could these be 'static' to this module? or maybe better, be 'static' in the route_notify_owner callback that uses them?